static/app/views/issueList/overview.tsx

```diff
@@ -139,7 +139,7 @@ interface EndpointParams extends Partial<PageFilters['datetime']> {
   useGroupSnubaDataset?: boolean;
 }
 
-type CountsEndpointParams = Omit<EndpointParams, 'cursor' | 'page' | 'query'> & {
+export type CountsEndpointParams = Omit<EndpointParams, 'cursor' | 'page' | 'query'> & {
```

Comment:
I'd either duplicate this type in `useFetchIssueCounts` or add it there and export here. Generally we don't want utils importing from components (should be the other way around)